### DISCUSSION TITLE
[Question] Inquiry Regarding Navier-Stokes Experiment Data Pair

### DISCUSSION THREAD

**Discussion by jiyeon914** [neuraloperator/neuraloperator#568] [Q&A]
I am a Ph.D. candidate conducting research on the prediction of turbulence dynamics using DL models. Currently, I am reproducing the results presented in the paper, but I have a question about the composition of data pairs.
Is it correct that one simulation becomes one input-output pair? In other words, in each case of Table 1 in the paper, is it correct to train one model with N ([0.10] input / (10,T] output) pairs?
I understood that by using 11 time steps in the channel direction and performing lifting-Fourier layers-projection, outputs for T-10 time steps can be obtained at once. However, although I understand the concept of the neural operator, a mapping between infinite-dimensional function spaces, I wonder if it is possible to predict the dynamics of flow fields for dozens of time steps using 11 inputs.
In fact, when I trained the model with the data I have (decaying 2D HIT without forcing), the prediction performance was too poor, which made me question whether I misunderstood the concept.

Thank you.

**Comment by arcturusannamalai** [comment]
Usually more layers can help if your loss function has plateaued.

**Comment by sunbingxin-jpg** [comment]
hello，could you please help me reproducing the results presented in the paper about ns ？

**Comment by dhpitt** [comment]
I think this is a better discussion than issue, since this refers to reproducing research not implemented in this repo.
